cli/src/command/wallet.rs

```diff
@@ -12,14 +12,26 @@ use log::LevelFilter;
 
 use crate::{error::Error, helper::get_password, println_log_info};
 
+const DEFAULT_NODE_URL: &str = "https://api.testnet.shimmer.network";
+const DEFAULT_WALLET_DATABASE_PATH: &str = "./stardust-cli-wallet-db";
+const DEFAULT_STRONGHOLD_SNAPSHOT_PATH: &str = "./stardust-cli-wallet.stronghold";
+
 #[derive(Debug, Clone, Parser)]
 #[command(author, version, about, long_about = None, propagate_version = true)]
 pub struct WalletCli {
-    #[command(subcommand)]
-    pub command: Option<WalletCommand>,
+    /// Set the path to the wallet database.
+    #[arg(long, value_name = "PATH", env = "WALLET_DATABASE_PATH", default_value = DEFAULT_WALLET_DATABASE_PATH)]
+    pub wallet_db_path: String,
+    /// Set the path to the stronghold snapshot file.
+    #[arg(long, value_name = "PATH", env = "STRONGHOLD_SNAPSHOT_PATH", default_value = DEFAULT_STRONGHOLD_SNAPSHOT_PATH)]
+    pub stronghold_snapshot_path: String,
+    /// Set the account to enter.
     pub account: Option<String>,
+    /// Set the log level.
     #[arg(short, long)]
     pub log_level: Option<LevelFilter>,
```

Comment:
We probably need to have somewhere in the docs where we present these parameters